docs/observability/getting-started.md

```diff
@@ -0,0 +1,89 @@
+# Get started with Genkit Monitoring
+
+This quickstart guide describes how to set up Firebase Genkit Monitoring for your deployed Genkit features, so that you can collect and view real-time telemetry data. With Firebase Genkit Monitoring, you get visibility into how your Genkit features are performing in production.
+
+Key capabilities include:
+
+* Viewing quantitative metrics like Genkit feature latency, errors, and token usage
+* Inspecting traces to see your Genkit's feature steps, inputs, and outputs, to help with debugging and quality improvement
+* Exporting production traces to run evals within Genkit
+
+Setting up Genkit Monitoring requires completing tasks in both your codebase and on the Google Cloud Console.
+
+## Before you begin
+
+1. If you haven't already, create a Firebase project. 
+
+    In the [Firebase console](https://console.firebase.google.com), click **Add a project**, then follow the on-screen instructions. You can create a new project or add Firebase services to an already existing Google Cloud project.
+2. Ensure your project is on the [Blaze pricing plan](https://firebase.google.com/pricing). 
+
+    Genkit Monitoring relies on telemetry data written to Google Cloud Logging, Metrics, and Trace which are paid services. View the [Google Cloud Observability](http://cloud/stackdriver/pricing#cloud-monitoring-pricing) page for pricing details and to learn about available free tier limits.
+3. Write a Genkit feature by following the [Get Started Guide](https://firebase.google.com/docs/genkit/get-started) and prepare your code for deployment by using one of the following guides:
+    1. [Deploy with Firebase](../firebase.md)
+    2. [Deploy with Cloud Run](../cloud-run.md)
+    3. [Deploy to any Node.js platform](../deploy-node.md)
+
+
+## Step 1. Add the Firebase plugin
+
+Install the `@genkit-ai/firebase` plugin in your project:
+
+```
+npm i –save @genkit-ai/firebase
+```
+
+Import `enableFirebaseTelemetry` into your Genkit configuration file (i.e. where `genkit(...)` is initalized) and call it:
+
+```typescript
+enableFirebaseTelemetry({
+  forceDevExport: false // set to true for local testing
```

Comment:
This doesn't need to be explicitly set -- it is off by default. Are we including it to make sure folks are aware it exists?

Reply:
I think so? @alexsinger